delira/data_loading/data_manager.py

```diff
@@ -245,108 +245,81 @@ def n_batches(self):
             raise ValueError('Invalid value for n_process_augmentation')
         return n_batches
 
-
-class ConcatDataManager(object):
+class ConcatDataManager(BaseDataManager):
```

Comment:
I don't quite get the use-case for the `ConcatDataManager`. The current implementation just concatenates all datasets with a `ConcatDataset` and then then merges them under the same `BaseDataManager`. For me, conceptually, `ConcatDataManager` should aggregate the `BaseDataManager` while retaining their initial configuration.

I.e. I have 2 Datasets (MRI + CT), and want to apply different transforms to each of them, therefore I setup 2 different `BaseDataManager` and combine them with `ConcatDataManager`, and use the result for training

Reply:
Currently the usecase is to wrap concatenate datasets which are already wrapped into `DataManager`s. 
IMO it does not make sense to keep the single managers and aggregate them, because there is no indexing performed and we would have to use additional samplers.
Additionally, it is (AFAIK) impossible to apply different transforms to subsets of the datamanager because the transforms are part of the `MultithreadedAugmenter` of `batchgenerators`.

Reply:
Okay, then I would argue that the cleaner solution is to aggregate the datasets once with `ConcatDataset` and instantiate a new `BaseDataManager` with the `ConcatDataset` and other desired parameters and remove the `ConcatDataManager` in a future release